packages/c/calamares/package.yml

```diff
@@ -17,36 +17,36 @@ summary    : Distribution-independent installer framework
 description: |
     Calamares is a distribution-independent system installer, with an advanced partitioning feature for both manual and automated partitioning operations. Calamares is designed to be customizable by distribution maintainers without the need for cumbersome patching, thanks to third-party branding and external modules support.
 builddeps  :
-    - pkgconfig(Qt5Core)
-    - pkgconfig(Qt5Quick)
-    - pkgconfig(Qt5Svg)
-    - pkgconfig(Qt5UiTools)
-    - pkgconfig(Qt5WebEngineWidgets)
+    - pkgconfig(Qt6Core)
+    - pkgconfig(Qt6Quick)
+    - pkgconfig(Qt6Svg)
+    - pkgconfig(Qt6UiTools)
+    - pkgconfig(Qt6WebEngineWidgets)
```

Comment:
I note  that it doesn't seem to be linking against Qt6-webengine or kf6-package. Are you sure you still need those deps?

Reply:
Let's find out! Rebuilding.